ui-tests/src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -1,4 +1,4 @@
-webapp.url=https://qa-refapp.openmrs.org/openmrs
+webapp.url=https://qa-refapp.openmrs.org/openmrs/
```

Comment:
Thanks @insookwa . This looks good, However you can remove the slash at the end of this url.

Reply:
i have removed it @sherrif10  thank you